src/main/java/org/isf/priceslist/manager/PriceListManager.java

```diff
@@ -42,97 +42,100 @@
 
 	@Autowired
 	private PricesListIoOperations ioOperations;
-	
+
 	/**
-	 * Return the list of {@link PriceList}s in the DB
-	 * @return the list of {@link PriceList}s
-	 * @throws OHServiceException 
+	 * Return the list of {@link List}s in the DB
+	 *
+	 * @return the list of {@link List}s
```

Comment:
This was fixed in https://github.com/informatici/openhospital-core/commit/947756e901c67d3b47ab2374857e28b015ef3644

Reply:
I believe I've resolved (merged) the changes from OP-399